### PR TITLE
feat: add default list styles option

### PR DESCRIPTION
**Pull request by nicolasiscoding**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,16 +90,26 @@ CSS list-style-type for `<ol>` element are now supported. Just do something like
...
+If you would like to override the default list-style-type, you would go pass in a parameter `defaultOrderedListStyleType` in the options object under the object `numbering`. For example:
+```
+  const options = {
+    numbering: {
+      defaultOrderedListStyleType: 'decimal'
+    }
+  }
+
+ const fileBuffer = await HTMLtoDOCX(htmlString, null, options);
+```
```
Comment (privateOmega) [inline README.md:112]: this should go in documentOptions under Parameters section.

File: src/utils/list.js [hunk 2]
```diff
@@ -1,42 +1,60 @@
...
+      case 'decimal':
+      default:
+        return `%${lvl + 1}.`;
+    }
   }
-};
+}
 
-export { getListStyleType, getListPrefixSuffix };
+export { ListStyleBuilder as default };
```
Comment (privateOmega) [inline src/utils/list.js:60]: why not use default export since its probably the only thing to be exported from this file

File: src/utils/list.js [hunk 3]
```diff
@@ -1,42 +1,60 @@
...
-      return 'decimal';
+class ListStyleBuilder {
+  // defaults is an object passed in from constants.js / numbering with the following properties:
+  // defaultOrderedListStyleType: 'decimal' (unless otherwise specified)
+  constructor(defaults) {
+    if (!defaults) {
+      throw new Error(
+        'ListStyleBuilder requires a defaults object. See constants.js / numbering for an example.'
+      );
+    }
```
Comment (privateOmega) [inline src/utils/list.js:9]: setup default value here, like

this.defaults = defaults || { defaultOrderedListStyleType: 'decimal' }.

It doesnt feel right throwing an error.

**Review by privateOmega — COMMENTED** [review]
@nicolasiscoding Thanks for the contribution

File: src/utils/list.js [hunk 4]
```diff
@@ -1,42 +1,54 @@
...
+      case 'upper-alpha-bracket-end':
+        return 'upperLetter';
+      case 'lower-alpha':
+      case 'lower-alpha-bracket-end':
+        return 'lowerLetter';
+      case 'decimal':
+      case 'decimal-bracket':
+      default:
+        return this.defaults.defaultOrderedListStyleType;
+    }
```
Comment (privateOmega) [inline src/utils/list.js:25]: this looks wrong

for case 'decimal' and 'decimal-bracket' it should be 'decimal' and for the case where its not explicitly specified, it should use `this.defaults.defaultOrderedListStyleType`

**Comment by nicolasiscoding** [comment]
Good catch

Sent via Superhuman iOS ***@***.***>

File: src/utils/list.js (hunk 4 above)
Comment (nicolasiscoding) [reply src/utils/list.js]: Done and rebased

**Review by privateOmega — APPROVED** [review]
LGTM

**Pull request merged**